modules/service/src/main/scala/lucuma/odb/service/UserInvitationService.scala

```diff
@@ -141,7 +141,7 @@ object UserInvitationService:
                   val xa = transaction
                   xa.savepoint.flatMap: sp =>
                     session
-                      .prepareR(ProgramService.Statements.LinkUser.command)
+                      .prepareR(ProgramUserService.Statements.LinkUser.command)
```

Comment:
Here in the `UserInvitationService` when a user accepts an invitation, we attempt to "link" them to the program (i.e., add them as a `ProgramUser`).  If previously added explicitly via `addProgramUser` then the linking part will fail with a warning.  I think we'll want to handle this more smoothly.